src/components/DatePicker/DatePicker.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Meta, StoryFn } from '@storybook/react';
+import { withDesign } from 'storybook-addon-designs';
+import { DatePicker } from './DatePicker';
+
+// More on default export: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
+export default {
+  title: 'Components/DatePicker',
+  component: DatePicker,
+  // More on argTypes: https://storybook.js.org/docs/react/api/argtypes
+  argTypes: {
+    onChange: { action: true },
+  },
+  args: {
+    label: 'Label',
+    width: 300,
```

Comment:
Pistä näitä argseja samaan tapaan kun on Input tai Dropdown asetettu, että property taulukossa olis arvoja esiasetettu laajemmin.